### DISCUSSION TITLE
Everytime I reboot my docker server I can no longer login to invoiceshelf

### DISCUSSION THREAD

**Discussion by kylelliott** [InvoiceShelf/InvoiceShelf#223] [General]
![IMG_0684](https://github.com/user-attachments/assets/6f243f4d-645f-4ec7-935a-ac7e7209b2c3)
![IMG_0683](https://github.com/user-attachments/assets/98e2f45e-7064-44b8-96e4-4e5e83cb5450)

**Comment by rihards-simanovics** [comment]
@kylelliott check the contents of the .env file inside the container and make sure that the values inside it still match the env variables you are using inside the docker manager. I find that on reboot the values get swapped for the different ones.
